interfaces/builtin/hardware_observe.go

```diff
@@ -29,12 +29,19 @@ const hardwareObserveConnectedPlugAppArmor = `
 # from the system.  this is reserved because it allows reading potentially sensitive information.
 # Usage: reserved
 
+# used by lscpu
```

Comment:
Originally I had separated this into another PR, but was asked to merge it into a single one:
https://github.com/snapcore/snapd/pull/1612
https://github.com/snapcore/snapd/pull/1618